.pytool/Plugin/CodeQL/MuCodeQlQueries.qls

```diff
@@ -4,50 +4,144 @@
 - queries: '.'
   from: codeql/cpp-queries
 
+##########################################################################################
 # "Core" Queries
+##########################################################################################
+
+## Errors
 - include:
-    id: cpp/badoverflowguard
+    id: cpp/pointer-overflow-check
+
+## Warnings
 - include:
     id: cpp/comparison-with-wider-type
 - include:
     id: cpp/conditionallyuninitializedvariable
 - include:
-    id: cpp/likely-bugs/memory-management/v2/conditionally-uninitialized-variable
+    id: cpp/uselesstest
+
+## Recommendations
 - include:
-    id: cpp/paddingbyteinformationdisclosure
+    id: cpp/redundant-null-check-param
+
+## Unknown
 - include:
-    id: cpp/pointer-overflow-check
+    id: cpp/badoverflowguard
 - include:
-    id: cpp/redundant-null-check-param
+    id: cpp/likely-bugs/memory-management/v2/conditionally-uninitialized-variable
 - include:
-    id: cpp/unguardednullreturndereference
+    id: cpp/paddingbyteinformationdisclosure
 - include:
-    id: cpp/uselesstest
+    id: cpp/unguardednullreturndereference
+
+##########################################################################################
+# Additional Fundamental Queries

```

Comment:
What's the criteria for "additional fundamental queries" that differentiate from "core queries" ? Can we get a blurb here like we have in "Extra Queries"?
Also what is the "##Unknown" category? Could it potentially produce an error or warning?

Reply:
The first distinction has to do with how query lists are treated internally. I can try to add more details, but it will be limited by that context.

The "Unknown" section came about because I couldn't initially find the severity level details of those after breaking them out, but I have that info now, so I'll move them to other sections.

Reply:
I removed the "Additional Fundamental Queries" section, converging its content with "Extra Queries" and moved "Unknown" queries to the appropriate severity level sections.